### PR TITLE
Update kubebench for v1alpha2

### PR DESCRIPTION
**Pull request by Akado2009**
Fixes: https://github.com/kubeflow/manifests/issues/111

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/manifests/114)

### CODE REVIEW

**Comment by kkasravi** [comment]
@Akado2009 you'll need to regenerate the unit-tests
1. cd manifests/tests
2. make generate
3. make test (verified they all PASS)
4. git commit -a

**Comment by Akado2009** [comment]
@kkasravi I see, thank you!

**Comment by xyhuang** [comment]
- the operator needs to mount the configmap and pass the config file in its commands.
- remove the dashboard stuff till the dashboard gets updated to catch up with v1alpha2. we don't want it to break the deployment for now.
- remove changes to jupyter which is non relevant to this commit.

File: kubebench/base/config-map.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+kind: ConfigMap
+metadata:
+  name: kubebench-config
+data:
+  kubebenchconfig.yaml: |
+    """
+    defaultWorkflowAgent:
+      container:
+        name: kubebench-workflow-agent
+        image: ciscoai/kubebench-workflow-agent:latest
```
Comment (hougangliu) [inline kubebench/base/config-map.yaml]: is this image official released? can we make it configurable by env or else?

File: kubebench/base/deployment.yaml [hunk 2]
```diff
@@ -32,6 +32,6 @@ spec:
         app: kubebench-operator
     spec:
       containers:
-      - image: gcr.io/kubeflow-images-public/kubebench/kubebench-operator:v0.4.0-13-g262c593
+      - image: gcr.io/kubeflow-images-public/kubebench/kubebench-operator-v1alpha2:v0.5.0-11-gea53ad5
```
[the remark this answers is not in the document]
Comment (Akado2009) [reply kubebench/base/deployment.yaml]: Fixed/

File: kubebench/base/config-map.yaml (hunk 1 above)
Comment (Akado2009) [reply kubebench/base/config-map.yaml]: I updated it to officially released

**Comment by Akado2009** [comment]
@xyhuang  How do I remove changes to jupyter? They are auto-generated, I haven't touched jupyter files.
Eveyrthing else is fixed I guess

**Comment by xyhuang** [comment]
> @xyhuang How do I remove changes to jupyter? They are auto-generated, I haven't touched jupyter files.
> Everything else is fixed I guess

try rebase to the current master?

**Comment by Akado2009** [comment]
@xyhuang done

File: kubebench/base/service-account.yaml [hunk 3]
```diff
@@ -1,15 +1,5 @@
 ---
 apiVersion: v1
 kind: ServiceAccount
-metadata:
-  name: kubebench-dashboard
----
-apiVersion: v1
-kind: ServiceAccount
 metadata:
   name: default
```
Comment (xyhuang) [inline kubebench/base/service-account.yaml]: this service account can be removed

File: tests/jupyter-web-app-overlays-application_test.go [hunk 4]
```diff
@@ -446,4 +446,4 @@ func TestJupyterWebAppOverlaysApplication(t *testing.T) {
 	}
 	expected, err := n.EncodeAsYaml()
 	th.assertActualEqualsExpected(m, string(expected))
-}
```
Comment (xyhuang) [inline tests/jupyter-web-app-overlays-application_test.go]: you might want to add a new line at the end.

**Comment by xyhuang** [comment]
sorry a few more nits... others look good

**Comment by Akado2009** [comment]
@xyhuang  finished

**Comment by ashahba** [comment]
Thanks @Akado2009 for this PR, once all the tests are regenerated and pass as per @kkasravi 's request it's ready to be merged.

**Comment by hougangliu** [comment]
Thanks
/lgtm
/approve

**Pull request merged**